src/rss/urls.py

```diff
@@ -6,7 +6,8 @@
 from rss import views
 
 urlpatterns = [
-    url(r'^', views.LatestArticlesFeed(), name='rss_articles'),
```

Comment:
A slight hiccough with this change: anyone who is subscribed to this URL will stop seeing new content.

Reply:
Ah, just reorder them so the LatestArticlesFeed one comes last in the list.

Reply:
Ah, it *does* work if I move it to the end of the URLs, I'll change that